editions/tw5.com/tiddlers/system/doc-styles.tid

```diff
@@ -189,12 +189,17 @@ a.doc-from-version.tc-tiddlylink {
 	fill: <<colour background>>;
     padding: 0 0.4em;
     font-size: 0.7em;
-    text-transform: uppercase;
     font-weight: bold;
     line-height: 1.5;
     vertical-align: text-bottom;
 }
 
+a.doc-from-version.doc-from-version-new.tc-tiddlylink {
+	background: #ffe246;
```

Comment:
This is not possible, since it needs to use a palette value. I also do not like the specificity of the selector. IMO it can be much less specific.

Reply:
I'll see how it looks with highlight or some other palette colors.
Side note: if we aim to base all docs styles on palette values only there is surely a lot of work in other places.